drivers/misc/coresight/nrf_etr.c

```diff
@@ -84,6 +443,43 @@ static void rd_idx_inc(void)
 	etr_rd_idx += CORESIGHT_TRACE_FRAME_SIZE32;
 }
 
+static void decode(uint8_t *buf, uint32_t pending)
```

Comment:
I'm curious why this decoder code is in an ETR driver. It seems to be a coresight-STM-related decoder, not an ETR-related decoder.